code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -0,0 +1,84 @@
+///////////////File for all snowflake/special races/////////////////////////////
+/////Anything that is spectacularly special should be put in here///////////////
+////////////////////////////////////////////////////////////////////////////////
+
+
+
+/datum/species/xenochimera //Scree's race.
+	name = "Xenochimera"
+	name_plural = "Xenochimeras"
+	icobase = 'icons/mob/human_races/r_tajaran.dmi' //Tajara basis, as it has working skin color.
+	deform = 'icons/mob/human_races/r_def_tajaran.dmi'
+	unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick, /datum/unarmed_attack/punch, /datum/unarmed_attack/bite)
+	num_alternate_languages = 2
+	secondary_langs = list("Sol Common")
+	color_mult = 1
```

Comment:
Tajaran sprites are meant for color_add, so you should remove this. Makes them really hard to color.